darwin/item.py

```diff
@@ -26,4 +31,5 @@ def parse_dataset_item(raw) -> DatasetItem:
         "n/a",
         raw["seq"],
         raw.get("current_workflow_id"),
+        raw["path"],
```

Comment:
@simedw is path always going to be there?

Reply:
I think we always give it in our current api / export format